<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kanbanapp</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/icons.qrc">
    <normaloff>:/res/icons/kanbanapp2.0_128.png</normaloff>:/res/icons/kanbanapp2.0_128.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="KanbanBoardWidget" name="boardWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_debug">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="title">
       <string>Debug</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_debug">
         <property name="text">
          <string>Debug - initialise second board</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KanbanBoardWidget" name="boardWidget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionNew_Board"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_List"/>
   <addaction name="actionMove_List_Left"/>
   <addaction name="actionMove_List_Right"/>
   <addaction name="actionDelete_List"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/opened_folder_48px.png</normaloff>:/res/icons/opened_folder_48px.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/icons8-save-48.png</normaloff>:/res/icons/icons8-save-48.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/code_48px.png</normaloff>:/res/icons/code_48px.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/icons8-save-as-48.png</normaloff>:/res/icons/icons8-save-as-48.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/info_48px.png</normaloff>:/res/icons/info_48px.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionAdd_List">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/add_list_48px.png</normaloff>:/res/icons/add_list_48px.png</iconset>
   </property>
   <property name="text">
    <string>Add List</string>
   </property>
   <property name="toolTip">
    <string>Add new list</string>
   </property>
  </action>
  <action name="actionMove_List_Left">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/open_pane_48px.png</normaloff>:/res/icons/open_pane_48px.png</iconset>
   </property>
   <property name="text">
    <string>Move List Left</string>
   </property>
   <property name="toolTip">
    <string>Move selected list left</string>
   </property>
  </action>
  <action name="actionMove_List_Right">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/open_pane_48px flipped.png</normaloff>:/res/icons/open_pane_48px flipped.png</iconset>
   </property>
   <property name="text">
    <string>Move List Right</string>
   </property>
   <property name="toolTip">
    <string>Move selected list right</string>
   </property>
  </action>
  <action name="actionDelete_List">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/delete_bin_48px.png</normaloff>:/res/icons/delete_bin_48px.png</iconset>
   </property>
   <property name="text">
    <string>Delete List</string>
   </property>
   <property name="toolTip">
    <string>Delete selected list</string>
   </property>
  </action>
  <action name="actionNew_Board">
   <property name="icon">
    <iconset resource="../res/icons.qrc">
     <normaloff>:/res/icons/new_file_48px.png</normaloff>:/res/icons/new_file_48px.png</iconset>
   </property>
   <property name="text">
    <string>New board</string>
   </property>
  </action>
  <action name="actionNo_recent_files">
   <property name="text">
    <string>No recent files</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KanbanBoardWidget</class>
   <extends>QWidget</extends>
   <header>kanbanboardwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/icons.qrc"/>
 </resources>
 <connections/>
</ui>
